### ISSUE TITLE
AugmentedImage - Detach from tracking

### ISSUE THREAD

**Issue by DavidArayan** [google-ar/arcore-android-sdk#651]
Hello,

It seems that AugmentedImage is continuously tracked once detected. This nicely covers the use case of "detect once, track forever" however what happens if we need to move the marker slightly to a new position?

Is the following possible?
1. Check if the AugmentedImage is currently in camera view. We have some math that does this, however it would be nice for this option to be standard as part of the AugmentedImage. Something like AugmentedImage.isVisible().
2. Stop tracking the AugmentedImage, forcing ARCore to only start tracking once the marker has been "found" again. Something like AugmentedImage.detach().

Thank you!

**Comment by Canato** [comment]
1. It should be checked all the time. If you cover your AugmentedImage, even in the same place, it stops to show.

Said that is good to understand that, today, ARCore is no build to be used in move targets/anchor/point clouds/ Images. Maybe in the future

**Comment by DavidArayan** [comment]
@Canato We need to be able to tell ARCore to stop tracking a marker if possible. At the moment once a marker starts tracking, it's impossible to get rid of it. But thank you for your input.

**Comment by inio** [comment]
Duplicate of #519
